packages/compass-aggregations/src/modules/explain.ts

```diff
@@ -204,7 +205,7 @@ export const explainAggregation = (): PipelineBuilderThunkAction<Promise<void>>
       }
     } catch (e) {
       // Cancellation is handled in cancelExplain
-      if (!dataService.isOperationCancelledError(e as Error)) {
```

Comment:
Can we keep this method? Feels kinda cumbersome that you need to depend on a separate library to be able to check that operation was canceled

Reply:
If using `isCancelError` is undesirable, I’d probably just make this `e?.name === 'AbortError'`, i.e. inline the method. That’s also what the DOM standard [uses in its example](https://dom.spec.whatwg.org/#aborting-ongoing-activities-example).

Reply:
I don't think it's undesirable, it's nice to have the method that is strictly typed and is checking for this, what is weird to me is that every place that needs to depend on data-service now also need to depend on compass-utils package

Reply:
Mh yeah, if that’s the concern re-exporting it from data-service is probably fine?

Reply:
I like both options, either have it:
1. exported from data-service
2. on data-service instance `ds.isCancelError(error: any): error is AbortError`. 

Slightly leaning towards latter as no extra import needed. Let me know what both think.

Reply:
Yeah, that seems fair and in line with the idea that we might have different DataService implementations at some point.

Reply:
Added in [`448adfd` (#3856)](https://github.com/mongodb-js/compass/pull/3856/commits/448adfdbabfb099b67adc1f4d279f4838855d8f5)